docs/how-to-guides/install-seven-add-on.md

```diff
@@ -0,0 +1,189 @@
+---
+myst:
+  html_meta:
+    "description": "How to install a Seven frontend add-on with Cookieplone"
+    "property=og:description": "How to install a Seven frontend add-on with Cookieplone"
+    "property=og:title": "Install Seven with Cookieplone"
+    "keywords": "Plone, Plone 6, Volto, create, project, install, Cookieplone"
+---
+
+
+(install-cookieplone-label)=
+
+# Install Seven with Cookieplone
+
+This chapter describes how you can create a web application using the {term}`Cookieplone` template.
+
+This template is the recommended way to start a new add-on using the Seven frontend.
+It also includes tools for development and deployment.
+
+```{seealso}
+For other installation options, see {ref}`get-started-install-label`.
+```
+
+(create-project-cookieplone-system-requirements)=
+
+## System requirements
+
+Plone and Seven have both hardware requirements and software prerequisites.
+
+
+### Supported web browsers
+
+```{include} /volto/_inc/_install-browser-reqs-volto.md
+```
+
+```{include} /_inc/_install-browser-reqs-classic-ui.md
+```
+
+
+(create-project-cookieplone-hardware-requirements-label)=
+
+### Hardware requirements
+
+```{include} /_inc/_hardware-requirements.md
+```
+
+
```

Comment:
This section should be placed before "Supported web browsers", to align with the order specified in line 28. We should also reorder this in Plone 6 documentation. This is my oversight. Coming at it with fresh eyes, it stuck out. I'll create a separate PR for that.

Reply:
Done in 5e437a39bfae9297babdf89e4399b8cf46f6ce83. See also https://github.com/plone/documentation/pull/1929.